packages/botonic-react/src/components/message.jsx

```diff
@@ -279,7 +281,7 @@ export const Message = props => {
               }}
               {...otherProps}
             >
-              {markdown ? (
+              {markdown && !isWhatsapp(requestContext) ? (
```

Comment:
should this be disabled for all providers except dev & webchat?
if we can never go here with providers like telegram, add comment explaining why